packages/react-next/RELEASE_NOTES.md

```diff
@@ -2,17 +2,9 @@
 
 ## Breaking changes
 
-### Beak
+### Calendar
 
-- Removed empty `IBeak` interface
-- Removed `componentRef` prop
-
-### SpinButton
-
-- Simplified props to `ISpinButtonStyles` to include only the parts of the component to bring inline with
-  other components.
-- Replaced `getClassNames` legacy prop with `styles` prop to bring component consistent to other components
-  and improve cachability of internal styles.
+TODO: Diff of OUFR vs date-time Calendar
```

Comment:
good call; this also reminded me, I have chatted with @lorejoh12 at one point related to why can't we just remove OUFR calendar and export and maintain only `date-time`. He did mention to me it's fully backward compatible (we should definitely double check), but the blocker is around untangling the dependencies (I'm not exactly sure how much work is needed, but we will run into this problem soon).

Reply:
I thought the plan for v8 was to remove the OUFR Calendar and DatePicker and re-export those from date-time. We need to clarify that.

Reply:
Hmm though I see what you mean by dependency issues... If we can't find a better approach, we could potentially move the date-time components into the main package and have date-time re-export them for now. Then move them back and fix the dependencies properly once individual-component packages are available.

Reply:
I made a note in the scope doc about looking into this, but in any case the decision shouldn't block this PR (since release notes aren't final).